src/e2e/testUtils.ts

```diff
@@ -55,50 +60,71 @@ export const createMockActorKey = async (
     type,
   });
 
-export const createWorldTestItem = async (type: string) =>
+export const createWorldTestItem = async (
+  type: string,
+  name: string = `New World Test ${type.capitalize()}`
+) =>
   Item.create({
     type,
-    name: `New World Test ${type.capitalize()}`,
+    name,
   });
 
 export const createActorTestItem = async (
-  actor: StoredDocument<Actor>,
-  type: string
-) =>
-  actor.createEmbeddedDocuments(
-    "Item",
-    [{ type, name: `New Actor Test ${type.capitalize()}` }],
-    {}
-  );
+  actor: StoredDocument<Actor> | undefined,
+  type: string,
+  name: string = `New Actor Test ${type.capitalize()}`,
+  data: object = {}
+) => actor?.createEmbeddedDocuments("Item", [{ type, name, ...data }]);
+
+export const createMockMacro = async () =>
+  Macro.create({
+    name: `Mock Macro ${foundry.utils.randomID()}`,
+    type: "script",
+    command: "console.log('Testing Macro');",
+  });
 
 export const createMockScene = async () =>
   Scene.create({ name: "Mock Scene", tokenVision: true });
 
+export const getMockActorKey = async (key: string) =>
+  game.actors?.getName(`Test Actor ${key}`);
+
 /**
  * CLEANUP HELPERS
  */
 
-export const cleanUpMacros = () => {
-  const mockMacros = game.macros?.filter((o) =>
-    o.name.includes("New Actor Test")
-  );
-  mockMacros?.forEach((o) => o.delete());
+export const cleanUpMacros = async () => {
+  const mockMacros = game.macros?.filter((o) => o.name?.includes("Mock Macro"));
+  mockMacros?.forEach(async (o) => await o.delete());
+  return true;
 };
 
-export const cleanUpActorsKey = (key) => {
+export const cleanUpActorsKey = async (key) => {
```

Comment:
name convention here: cleanUpActors*By*Key() ? Or just cleanUpActors? Not a bad name here just curious

Reply:
Didn't think of it before you said it, but I agree that the first version is better. Makes it clearer to what it actually does. I'm changing that!